src/views/ModelPlan/TaskList/Beneficiaries/Frequency/index.tsx

```diff
@@ -362,28 +369,75 @@ const Frequency = () => {
+                              </p>
+
+                              <FieldErrorMsg>
+                                {flatErrors.precedenceRules}
+                              </FieldErrorMsg>
+
+                              {getKeys(beneficiaryPrecedenceConfig.options).map(
+                                key => (
+                                  <Fragment key={key}>
+                                    <Field
+                                      as={CheckboxField}
+                                      id={`beneficiaries-precedence-rules-${key}`}
+                                      data-testid={`beneficiaries-precedence-rules-${key}`}
+                                      name={`precedence-rules-${key}`}
+                                      label={
+                                        beneficiaryPrecedenceConfig.options[key]
+                                      }
+                                      value={key}
+                                      checked={values.precedenceRules.includes(
+                                        key
+                                      )}
+                                      onChange={(
+                                        e: React.ChangeEvent<HTMLInputElement>
+                                      ) => {
+                                        if (e.target.checked) {
+                                          arrayHelpers.push(e.target.value);
+                                        } else {
+                                          const idx = values.precedenceRules.indexOf(
+                                            e.target.value as YesNoType
+                                          );
+                                          arrayHelpers.remove(idx);
+                                        }
+                                      }}
+                                    />
+
+                                    {values.precedenceRules?.includes(key) && (
+                                      <div className="margin-left-4">
+                                        <span>
+                                          {miscellaneousT('pleaseDescribe')}
+                                        </span>
+                                        <Field
+                                          as={TextAreaField}
+                                          className="height-15"
+                                          error={flatErrors.precedenceRules}
+                                          id={`beneficiaries-precedence-rules-${key}-note`}
+                                          data-testid={`beneficiaries-precedence-rules-${key}-note`}
+                                          name={`precedenceRules${beneficiaryPrecedenceConfig.options[key]}`}
+                                        />
+                                      </div>
+                                    )}
+                                  </Fragment>
```

Comment:
This component needs to be it's mapping in the translation file, just like the parent.  It shouldn't be run through the miscellaneous translation library